src/commands/buildCommands/config/webpack/browserAppConfig.js

```diff
@@ -137,7 +137,16 @@ function getConfig(localeCode) {
 		injectHotReloadConfig(config);
 	}
 	if (env.properties.isProd) {
-		config.plugins = config.plugins.concat(prodPlugins);
+		config.plugins = config.plugins.concat(
+			prodPlugins,
+			new SWPrecacheWebpackPlugin({
+				cacheId: 'mwp',
+				dontCacheBustUrlsMatching: /\.\w{8}\./,
```

Comment:
I copied it from the recommended config, but looking into [`dontCacheBustUrlsMatching`](https://github.com/GoogleChrome/sw-precache#dontcachebusturlsmatching-regex), it's not working with our hashed asset filenames currently - I'll update and add code comments to explain